avm/res/cache/redis-enterprise/tests/e2e/kv-secrets/dependencies.bicep

```diff
@@ -0,0 +1,21 @@
+@description('Optional. The location to deploy to.')
+param location string = resourceGroup().location
+
+@description('Required. The name of the Managed Identity to create.')
+param keyVaultName string
+
+resource keyVault 'Microsoft.KeyVault/vaults@2021-06-01-preview' = {
+  name: keyVaultName
+  location: location
+  properties: {
+    sku: {
+      family: 'A'
+      name: 'standard'
+    }
+    enableRbacAuthorization: true
+    tenantId: subscription().tenantId
```

Comment:
```suggestion
    tenantId: subscription().tenantId
    enablePurgeProtection: null
```

Not sure, is `purgeProtection` not enabled by default? If so, we should definitely disable it. The same comment would actually go for the `redis` module update

Reply:
It's disabled by default.

![image](https://github.com/user-attachments/assets/031342e9-c587-4792-862e-792ccc9ac0c7)